nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/util/SFTPTransfer.java

```diff
@@ -72,17 +74,65 @@
 import java.text.DateFormat;
 import java.text.SimpleDateFormat;
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
 import java.util.Date;
 import java.util.List;
 import java.util.Locale;
 import java.util.Set;
 import java.util.concurrent.TimeUnit;
 import java.util.regex.Pattern;
+import java.util.stream.Collectors;
 
 import static org.apache.nifi.processors.standard.util.FTPTransfer.createComponentProxyConfigSupplier;
 
 public class SFTPTransfer implements FileTransfer {
+    // The following DefaultConfig is used for configuration purposes. There are PropertyDescriptors associated with this class
+    // that allow an operator to choose various algorithms and ciphers to allow to be used for their SSH connections.
+    private static final DefaultConfig DEFAULT_CONFIG = new DefaultConfig();
+
+    private static final Set<String> AVAILABLE_KEY_ALGORITHM_NAMES_SET = Collections.unmodifiableSet(DEFAULT_CONFIG.getKeyAlgorithms().stream()
```

Comment:
Recommend removing the `_SET` suffix from this and other variable names.  In light of the values coming from `DefaultConfig`, it seems like it would be more accurate to use `DEFAULT` instead of `AVAILABLE`.
```suggestion
    private static final Set<String> DEFAULT_KEY_ALGORITHM_NAMES = Collections.unmodifiableSet(DEFAULT_CONFIG.getKeyAlgorithms().stream()
```

Reply:
Done.